src/components/timeline/timeline.js

```diff
@@ -60,13 +63,15 @@ const Timeline = ( { revisions, users } ) => {
 				</div>
 			</div>
 			{edits}
+			{spinner}
 		</div>
 	);
 };
 
 Timeline.propTypes = {
 	users: PropTypes.instanceOf( Set ).isRequired,
-	revisions: PropTypes.instanceOf( Map ).isRequired
+	revisions: PropTypes.instanceOf( Map ).isRequired,
+	status: PropTypes.string.isRequired
```

Comment:
Since this is an enum, use something like:
```
status: PropTypes.oneOf( [ 'fetching', 'ready' ] ).isRequired
```

Reply:
Done